modules/packages/msiexec.bat

```diff
@@ -96,11 +96,15 @@ rem Install this file if it exists
     goto :EOF
   )
 
-  REM TODO: ENT-6824 save this logfile based on msi filename
-  set logfile="\cfengine_package_install.log"
-  %MSIEXEC% /quiet /passive /qn /norestart /l*vx %logfile% /i %1
+  set log_dir="\cfengine_package_logs\"
+  if not exist %log_dir% (
+    mkdir %log_dir%
+  )
+  for /F "delims=" %%i in (%1) do @set basename="%%~ni"
+  set log_file="%log_dir:"=%%basename:"=%_install.log"

```

Comment:
replace quotes with nothing because otherwise I end up with two double quotes together which breaks the path (i think)

Reply:
might be worth an inline comment `REM`

Reply:
+1